src/components/TrackTile/hooks/useAxiosTrackTileService.ts

```diff
@@ -28,6 +28,7 @@ import { pick, merge, pickBy, omit, fromPairs } from 'lodash';
 import { useHttpClient } from '../../../hooks/useHttpClient';
 import { useActiveAccount } from './../../../hooks/useActiveAccount';
 import { useActiveProject } from './../../../hooks/useActiveProject';
+import { useUser } from '../../../hooks';
```

Comment:
Nit-pick: ideally we wouldn't import from index files like this.  It creates a scenario where, if someone wants to import this service but does NOT want to import another hook, an SDK consumer is out of luck.  If we only import non-index-files, it leaves the door open to avoid hooks/components/etc. an SDK consumer is having trouble with.